### DISCUSSION TITLE
Preview Environments for Plausible?

### DISCUSSION THREAD

**Discussion by evoxmusic** [plausible/analytics#3386] [Anything Else]
Hey Plausible team 👋🏽 ,

I’m Romaric from the [Qovery](https://github.com/qovery) team. 

I’m a huge believer in open-source and absolutely love the work you and your team have put into Plausible. It’s projects like yours that keep the tech world spinning! So congrats!

[I’ve kickstarted a Support Open Source Program for Qovery Preview Environments](https://www.qovery.com/blog/free-preview-environments-for-open-source-projects). In a nutshell, Preview Environments can help you to:

- **Test in Real Conditions**: With an isolated environment for each pull request, contributors can thoroughly test their changes in real-life conditions before the code is merged.
- **Accelerate Feedback Loop**: It allows the community to interact with the changes early on and provide valuable feedback, leading to better and faster improvements.
- **Bug Detection**: The chance of bugs sneaking into the production code is significantly reduced, which means less time debugging and more time building cool features!
- **Collaborative Spirit**: It fosters a collaborative spirit within the community by enabling contributors to see the direct impact of their changes and encourages new contributors to participate in the project.

> If you are curious to see how it works, take a look at the end of [my article](https://www.qovery.com/blog/free-preview-environments-for-open-source-projects).

If this piques your interest, please drop me a line with a good time to chat, or sign up and try it yourself. It's 100% free for Plausible.

Keep up the fantastic work :muscle:

Romaric.

**Comment by cnkk** [comment]
Hey @evoxmusic,

Thanks for getting in touch!

We have considered spinning up preview environments for PRs in the past, but our current workflow works fine without it. So we don't have a need for it right now.

But we will definitely look into Qovery if the topic comes up again :)

**Reply by evoxmusic** [reply]
Thank you for getting back to me @cnkk 🙏
